src/components/ToolBar/Dropdown/Dropdown.js

```diff
@@ -1,14 +1,38 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import SearchResults from '../Search/DropdownList.js';
 import * as styles from '../Search/search.module.css';
 import AddEquipment from '../../AddEquipment/AddEquipment';
 import arrow from '../../../images/hideIcon.png';
 
 const Dropdown = ({ data, currEquip }) => {
     const [hidden, setHide] = useState(true);
+    const [iosWithNotch, setIosWithNotch] = useState(false);
+
+    const isIPhoneWithNotch = () => {
+        if (typeof window !== 'undefined' && typeof navigator !== 'undefined') {
+            let iHeight = window.screen.height;
+            let iWidth = window.screen.width;
+
+            if (
+                iWidth >= 375 &&
+                iHeight >= 812 &&
+                /iPhone/.test(navigator.userAgent) &&
+                !window.MSStream
+            )
+                return true;
+        }
+    };
+
+    useEffect(() => {
+        setIosWithNotch(isIPhoneWithNotch());
+    }, []);
```

Comment:
Винести в хук `useNotchDetector()`

```suggestion
  const iosWithNotch = useNotchDetector();
```

Логіка перевірки розмірів, стан влагу та ефект мають перенестися в кастомний хук `useNotchDetector`. Хук повертає флаг, чи має телефон провку.